bindings/js/templates/main.tmpl

```diff
@@ -0,0 +1,26 @@
+/**
+    This binding file was auto generated based on FLIX template v{{.Version}}. 
+    Changes to this file might get overwritten 
+**/
+
+import * as fcl from "@onflow/fcl"
```

Comment:
We should have a type (.e.g `fcl` as well) so that other libraries can add their bindings and it's not FCL specific

Reply:
not following, can you provide an example?